org-cyf-sdc/hugo.toml

```diff
@@ -24,17 +24,22 @@ baseURL = "https://sdc.codeyourfuture.io/"
 [params]
 googleFonts="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300;500;700&family=Inter:wght@400;600&display=swap"
 
+#this makes the section nav go the right way
+# because of this 'unexpected behaviour' https://gohugo.io/methods/page/nextinsection/
```

Comment:
Can this one not be inherited from theme? 

Actually maybe this should be tidied up a bit more. What can we inherit from org-cyf-theme module?

https://gohugo.io/getting-started/configuration/#configure-page